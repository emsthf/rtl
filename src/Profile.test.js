import React from "react";
import { render } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile />", () => {
  it("matchers snapshot", () => {
    const utils = render(<Profile username="emsthf" name="든솔" />);
    expect(utils.container).toMatchSnapshot();
  });

  it("show the props correctly", () => {
    const utils = render(<Profile username="emsthf" name="든솔" />);
    utils.getByText("emsthf");
    // utils.getByText("든솔");
    utils.getByText(/든/);
  });
});
