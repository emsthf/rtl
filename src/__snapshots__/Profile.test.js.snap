// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matchers snapshot 1`] = `
<div>
  <div>
    <b>
      emsthf
    </b>
     
    <span>
      (
      든솔
      )
    </span>
  </div>
</div>
`;
